pressbooks/pressbooks-book-directory-fe
# Pb 61 increase e2e tests

- Adding more E2E tests with basic features: https://github.com/pressbooks/pressbooks-book-directory-fe/issues/61
- Fixing some issues with new filters: https://github.com/pressbooks/pressbooks/issues/1982

### How to test
- Run tests locally with Chrome and Firefox browsers: `npm run e2e -- --env firefox,chrome`. (Note: I could takes ~5 mins). All assertions (587) should passes.
- You can see reports locally running `npm run nightwatch:output` and open .html file in your browser located in `e2e_tests/output/output.html`.
